packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -170,6 +174,16 @@ export const CollectionScreenDetailsTile = ({
 
   const isReachable = useSelector(getIsReachable)
 
+  const { data: currentUserId } = useGetCurrentUserId({})
+  const { data: collection } = useGetPlaylistById(
+    {
+      playlistId: typeof collectionId === 'number' ? collectionId : -1,
+      currentUserId
+    },
+    { disabled: typeof collectionId !== 'number' }
+  )
+  const isStreamGated = collection?.is_stream_gated
```

Comment:
nit: This won't always resolve to a boolean?

Reply:
wdym? `collection` could be `undefined` at this point

Reply:
Right, so `isStreamGated` wouldn't be true/false, it'll be undefined.